### PR TITLE
Dat257 75

### PR DESCRIPTION
**Pull request by Huffla**
Fixed issues with year-month-day being shown.
Rounded values so that they do not stick out of the graph.

### CODE REVIEW

File: Client/Components/ComparisonComponent.razor [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
     <RadzenChart>
+           
         <RadzenBarSeries Fill="@Colors.Warning" Data="@OriginValueList" CategoryProperty="Date" Title="@CountryOrigin.Name" LineType="LineType.Dashed" ValueProperty="Value">
             <RadzenSeriesDataLabels Visible="true" />
         </RadzenBarSeries>
         <RadzenBarSeries Data="@ComparisonValueList" CategoryProperty="Date" Title="@CountryComparison.Name" LineType="LineType.Dashed" ValueProperty="Value">
             <RadzenSeriesDataLabels Visible="true" />
         </RadzenBarSeries>
-        <RadzenValueAxis Min="0" Max="@SetBarWidth()">
+            <RadzenValueAxis Min="0" Max="@SetBarWidth()" FormatString="{0:N}" >
```
Comment (jakobschy) [inline Client/Components/ComparisonComponent.razor]: Remove Indentation

File: Client/Components/ComparisonComponent.razor [hunk 2]
```diff
@@ -1,20 +1,28 @@
...
+
```
Comment (jakobschy) [inline Client/Components/ComparisonComponent.razor]: Remove New Lines

**Review by jakobschy — DISMISSED** [review]
Remove added Indentation and New Lines in ComparisonComponent.razor

**Comment by Flaskhals68** [comment]
Looks mostly good. 
Integer overflows are still unhandled and there are still some graphs where the labels are overflowing and covering parts of the courties names.

image

**Comment by Flaskhals68** [comment]
Should the unit not be displayed where it says "Consumption"?

File: Client/Components/ComparisonComponent.razor (hunk 2 above)
Comment (Huffla) [reply Client/Components/ComparisonComponent.razor]: Done

File: Client/Components/ComparisonComponent.razor [hunk 3]
```diff
@@ -1,20 +1,25 @@
 ﻿
 
 <div class="main">
+    <div>
     <h4>@CountryComparison.Name uses @GetComparisonPercentage(ComparisonCountryValue, OriginCountryValue) @ResourceType than @CountryOrigin.Name </h4>
     <p>Unit: @Unit</p>
     <RadzenChart>
+
```
Comment (filipkauffeldt) [inline Client/Components/ComparisonComponent.razor]: Remove new line

File: Client/Components/ComparisonComponent.razor [hunk 4]
```diff
@@ -1,20 +1,25 @@
 ﻿
 
 <div class="main">
+    <div>
```
Comment (filipkauffeldt) [inline Client/Components/ComparisonComponent.razor]: Should probably indent content in `div` to keep consistent code style

**Comment by Huffla** [comment]
Fixed issues with absurd negative numbers, and formatted the numbers in a way that looks better, rounded them to a few decimal places.
Switches position of the unit to where consumption is.
Changed colors of the comparison cards to better match the color scheme that we chose for the project.
Also changed so when a values is close to 0 the comparison is not in percent but instead in the amount of units.

File: Client/Components/Compare.razor.css [hunk 5]
```diff
@@ -97,13 +97,18 @@
 .grid-item {
     background-color: #f0f0f0;
     min-height: 25vh;
-    max-height: 60vh;
+    max-height: 70vh;
     text-align: center;
     border-radius: 40px;
-    box-shadow: 0 0 15px 1px #808080;
+    box-shadow: 0 0 15px 1px #057dcd;
```
Comment (Flaskhals68) [inline Client/Components/Compare.razor.css:103]: This feel a bit distracting.
Maybe keep the original border and box-shadow.

File: Client/Components/DashBoard.razor.css [hunk 6]
```diff
@@ -66,13 +66,18 @@
 .grid-item {
     background-color: #f0f0f0;
     min-height: 25vh;
-    max-height: 60vh;
+    max-height: 70vh;
     text-align: center;
     border-radius: 40px;
-    box-shadow: 0 0 15px 1px #808080;
+    box-shadow: 0 0 15px 1px #057dcd;
```
Comment (Flaskhals68) [inline Client/Components/DashBoard.razor.css]: This feel a bit distracting.
Maybe keep the original box-shadow and border?

**Review by Flaskhals68 — APPROVED** [review]
Looks great!

**Pull request merged**